infrastructure-tests/runTests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh -e
+
+# Sanity check
+if [ -z "$PPIPER_INFRA_IT_CF_USERNAME" ]; then
+    echo "Failure: Variable PPIPER_INFRA_IT_CF_USERNAME is unset"
+    exit 1
+fi
+
+if [ -z "$PPIPER_INFRA_IT_CF_PASSWORD" ]; then
+    echo "Failure: Variable PPIPER_INFRA_IT_CF_PASSWORD is unset"
+    exit 1
+fi
+
+set -x
+
+# Start a local registry, to which we push the images built in this test, and from which they will be consumed in the test
+docker run -d -p 5000:5000 --restart always --name registry registry:2 || true
+find ../cx-server-companion -type f -exec sed -i "" -e 's/ppiper/localhost:5000\/ppiper/g' {} \;
+
+# Configure our testing Jenkins instance via Configuration as Code (Create build job, register shared libraries, configure executors)
+mkdir -p ../cx-server-companion/life-cycle-scripts/jenkins-configuration
+cp testing-jenkins.yml ../cx-server-companion/life-cycle-scripts/jenkins-configuration
+
+docker build -t localhost:5000/ppiper/container-structure-test:latest ../container-structure-test
+docker build -t localhost:5000/ppiper/jenkins-master:latest ../jenkins-master
+
+# The cx-server script will be updated to the version where this URI points to.
+#If you need to change it in a PR to work, change it here temporarily and move it afterwards back to upstream.
+docker build --build-arg cx_server_base_uri=https://raw.githubusercontent.com/sap/devops-docker-images/master/jenkins-master/cx-server/cx-server -t localhost:5000/ppiper/cx-server-companion:latest ../cx-server-companion
```

Comment:
Do we still need this build arg. I thought we get rid of it...

Reply:
Well, it is all fine if your PR does not change cx-server, but if it does, you need a way to not overwrite it with the published file, ight

Reply:
The build.arg isn't even an build arg anymore:
https://github.com/SAP/devops-docker-images/blob/master/cx-server-companion/Dockerfile

I will always pick the files located in life-cycle-scripts/*